monkey/monkey_island/cc/services/initialize.py

```diff
@@ -40,4 +45,28 @@ def initialize_services(data_dir: Path) -> DIContainer:
 
 def _build_agent_binary_repository():
     file_repository = LocalStorageFileRepository(AGENT_BINARIES_PATH)
-    return AgentBinaryRepository(file_repository)
+    agent_binary_repository = AgentBinaryRepository(file_repository)
+
+    _log_agent_binary_hashes(agent_binary_repository)
```

Comment:
We don't need to log hashes to `_build_agent_binary_repository`. I think `_log_agent_binary_hashes` should be called in `initialize_services` once the repository is built